starfish/test/spots/detector/test_spot_detection.py

```diff
@@ -41,69 +49,31 @@ def simple_local_max_spot_detector() -> LocalMaxPeakFinder:
 trackpy_local_max_spot_detector = simple_trackpy_local_max_spot_detector()
 local_max_spot_detector = simple_local_max_spot_detector()
 
-
-def synthetic_two_spot_3d_2round_2ch() -> ImageStack:
-    """produce a 2-channel 2-round ImageStack
-
-    Notes
-    -----
-    - After Gaussian filtering, all max intensities are 7
-    - Two spots are located at (4, 10, 90) and (6, 90, 10)
-    - Both spots are 1-hot, and decode to:
-        - spot 1: (round 0, ch 0), (round 1, ch 1)
-        - spot 2: (round 0, ch 1), (round 1, ch 0)
-
-    Returns
-    -------
-    ImageStack :
-        noiseless ImageStack containing two spots
-
-    """
-
-    # blank data_image
-    data = np.zeros((2, 2, 10, 100, 100), dtype=np.float32)
-
-    # round 0 channel 0
-    data[0, 0, 4, 10, 90] = 1.0
-    data[0, 0, 5, 90, 10] = 0
-
-    # round 0 channel 1
-    data[0, 1, 4, 10, 90] = 0
-    data[0, 1, 5, 90, 10] = 1.0
-
-    # round 1 channel 0
-    data[1, 0, 4, 10, 90] = 0
-    data[1, 0, 5, 90, 10] = 1.0
-
-    # round 1 channel 1
-    data[1, 1, 4, 10, 90] = 1.0
-    data[1, 1, 5, 90, 10] = 0
-
-    data = gaussian_filter(data, sigma=(0, 0, 2, 2, 2))
-    return ImageStack.from_numpy_array(data)
-
-
-# create the data_stack
-data_stack = synthetic_two_spot_3d_2round_2ch()
-
-
-@pytest.mark.parametrize('data_stack, spot_detector, radius_is_gyration', [
-    (data_stack, gaussian_spot_detector, False),
-    (data_stack, trackpy_local_max_spot_detector, True),
-    (data_stack, local_max_spot_detector, False)
-])
+# test parameterization
+test_parameters = (
+    'data_stack, spot_detector, radius_is_gyration, max_intensity',
+    [
+        (ONE_HOT_IMAGESTACK, gaussian_spot_detector, False, ONE_HOT_MAX_INTENSITY),
```

Comment:
It looks like the radius_is_gyration parameter only gets used in the `reference_image is not None` case, so testing every scenario for each case seems like overkill?

Reply:
I'm going to be seriously revisiting spot detection soon (issues to come today AM). Trackpy will likely be removed, and this (good) point will be mooted.